src/oracles/RedstonePriceFeedWithRounds.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.17;
+
```

Comment:
if this is going to live in the repo here it needs comments & natspec

Reply:
@chrismaree I pulled this out to https://github.com/UMAprotocol/redstone-contracts/tree/master/src/examples